### PR TITLE
Removal of sonatype-work

### PR DESCRIPTION
**Pull request by joedragons**
Work on https://issues.sonatype.org/browse/NEXUS-9223 - removal of sonatype-work references from NX3 documentation.

Bamboo: http://bamboo.s/browse/BOOK-NEXUS44

### CODE REVIEW

File: chapter-installation.asciidoc [hunk 1]
```diff
@@ -149,18 +149,6 @@ to run Nexus as a specific user you could install into the
 +AppData\Local+ directory of that users home directory. Otherwise
 simply use e.g., +C:\nexus+ or something similar.
 
-The Nexus application directory +nexus-3.0.0-b2015061001+ requires
-another directory named +sonatype-work+. This directory contains all
-of the repository and configuration data for Nexus and is stored
```
Comment (mosabua) [inline chapter-installation.asciidoc]: Should be reworded to talk about the nested data folder instead and then about how to configure it to be somewhere else..

Comment (joedragons) [inline chapter-installation.asciidoc]: @mosabua Won't that be covered in the blobstore section?  Or maybe I'm missing what you want here.

**Comment by joedragons** [comment]
I see some references in chapter-migrating.asciidoc but it does not look like that's live in NX3 yet.  Rather than blindly stripping or rewriting that chapter I am leaving those alone.

File: chapter-installation.asciidoc (hunk 1 above)
Comment (mosabua) [inline chapter-installation.asciidoc]: I am updating this as well as reviewing and merging and publishing of the whole enchilada ;-)

File: chapter-introduction.asciidoc [hunk 2]
```diff
@@ -233,9 +233,7 @@ a single repository group, and you should download a copy of the free
 Nexus book - 'Repository Management with Nexus'. Once you've
 familiarized yourself with {oss}, you can easily upgrade to
 {pro} by downloading and installing Nexus
-Pro. Nexus stores all of your repository data and
-configuration in a directory named +sonatype-work+, which is separate
```
Comment (mosabua) [inline chapter-introduction.asciidoc]: Just fyi .. that chapter is not yet included and might never be ;-)

**Pull request merged**